packages/dmn-editor/tests/e2e/drgElements/changeBKMProperties.spec.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { test, expect } from "../__fixtures__/base";
+import { DataType } from "../__fixtures__/jsonModel";
+import { DefaultNodeName, NodeType } from "../__fixtures__/nodes";
+
+test.beforeEach(async ({ editor }) => {
+  await editor.open();
+});
+
+test.describe("Change Properties - BKM", () => {
+  test.beforeEach(async ({ palette, nodes, propertiesPanel }) => {
+    await palette.dragNewNode({ type: NodeType.BKM, targetPosition: { x: 100, y: 100 } });
+    await nodes.select({ name: DefaultNodeName.BKM });
+    await propertiesPanel.open();
+  });
+
+  test("should change the BKM node name", async ({ nodes, generalProperties }) => {
+    await generalProperties.changeNodeName({ from: DefaultNodeName.BKM, to: "Renamed BKM" });
+
+    await expect(nodes.get({ name: "Renamed BKM" })).toBeVisible();
```

Comment:
We need to make an assertion on the properties panel too.

Reply:
done in #27